### ISSUE TITLE
Add a fold and expand button at the top of the document

### ISSUE THREAD

**Issue by 5kyfkr** [Macavity/siyuan-database-properties-panel#14]
**Describe the idea**
The plugin is very useful. I hope to add the function 2 in the following link, which can fold and expand the view. This link also contains the final optimization plan for the property panel that I discussed at that time. You can refer to it to see how much can be completed. Now the developers of siyuan are really not energetic enough. It would be great if you can do it here.
https://github.com/siyuan-note/siyuan/issues/11371#issuecomment-2115626285

[Function point 2] Add a fold and expand button at the top of the document (alternative solution), or integrate the fold and expand button into the database logo at the top of the existing document (recommended solution). Click the database logo at the top of the document to expand or collapse the database property interface, and provide setting options to set the default expansion or default folding

**Comment by Macavity** [comment]
There is a lot of different discussion going on in the linked issue and I'll be honest, it's too much too read for me.

I think there is a reasonable use case to allow to fold/expand the database properties, but for my use-case it's not something I think is required or necessary.

If you have an idea what it could look like and we come do an agreement, I would be very happy to see this contribution being added to the plugin 😁

**Comment by 5kyfkr** [comment]
Yes, the most important thing at present is to make the editing function available. Another thing is to switch between groups to display the properties of different databases, which can reduce the space occupied by the property view at the top of the note.
The hiding function can make it easier for people to decide which documents do not display database properties, so as to maintain the current usage habits to a greater extent. Of course, this function has a lower priority.

**Comment by Achuan-2** [comment]
I hope that if the document has not been added to the database, the collapse button should not be displayed

**Comment by Achuan-2** [comment]
I am currently unclear about the reason; among all the documents not added to the database, some documents have fold/expand buttons, while others do not

**Comment by Macavity** [comment]
I created this bug: https://github.com/Macavity/siyuan-database-properties-panel/issues/38. Please don't comment on closed issues, I won't see the updates.

**Comment by Macavity** [comment]
The issue has been fixed and the Marketplace version 1.1.1 fixes the issue.
